### ISSUE TITLE
How to setup upload only monitoring?

### ISSUE THREAD

**Issue by victoriusnet** [abraunegg/onedrive#402]
Hi guys!

I could not figure it out how can I set to monitor one folder on my system if there is a new file / folder upload it, but never download anything from OneDrive.

Could you please help me? In this thread: https://github.com/abraunegg/onedrive/issues/356 I've read you are modifying entrypoint.sh, but how? I tried to change the ARGs but if I delete any file on my server it disappearing from the OneDrive too.

I've managed to setup selective sync to my folder called webbackup in ~/OneDrive/webbackup

I've stopped onedrive service then run: **onedrive --synchronize --upload-only --no-remote-delete** command it was OK.

But if I run command with monitoring enabled: **onedrive --monitor --synchronize --upload-only --no-remote-delete**  it deletes the file from OneDrive if I delete from local.

So as I summarize: I would like to monitor my webbackup folder and if there is any new file / folder I would like to upload it to OneDrive, but don't want to download or delete any file / folder from OneDrive.

How can I do that?

Thank you for your help!

**Comment by abraunegg** [comment]
@victoriusnet 
Essentially your trying to use flags that can't be used in the manner your thinking or would like.

All the extra flags & switches are only applicable for `--synchronize` and are only applicable for `--monitor` if specified.

Additionally, you cannot run `--monitor --synchronize` together - monitor will take precedence & all your flags / switches will be dropped - which is what your seeing the undesirable actions occurring.

To achieve what your wanting to do:
* Do not use `--monitor`
* Schedule a cronjob every X minutes to sync
* Use the correct application parameters for your use case: `--confdir '<your conf dir>' --synchronize --verbose -upload-only --no-remote-delete`

In short, you cannot use `--monitor` to achieve your desired outcome.

**Comment by abraunegg** [comment]
Closing issue as question answered
